CONTRIBUTING.md

```diff
@@ -160,27 +168,43 @@ Uninstalling keras-nightly-2.5.0.dev2021032500:
 
 ## Run tests
 
+We use [Bazel](https://bazel.build/) to build and run the tests.
+
+### Run a test file
+
+For example, to run the tests in `keras/engine/base_layer_test.py`,
+we can run the following command at the root directory of the repo.
+
+```shell
+bazel test keras/engine:base_layer_test
+```
+
+`keras/engine` is the relative path to the directory containing the test file.
+`base_layer_test` is the test file name without the `.py` extension.
+
+### Run a single test case
+
+The best way to run a single test case is to comment out the rest of the test
```

Comment:
I still think the --test_filter should work for bazel. let me have a try.

Reply:
If `test_filter` doesn't work, then an alternative method to commenting things out is to globally rename the tests to add some prefix like 'NORUN_' to the method name.